src/PageItem.js

```diff
@@ -52,19 +52,13 @@ PageItem.displayName = 'PageItem';
 export default PageItem;
 
 function createButton(name, defaultValue, label = name) {
-  return class extends React.Component {
-    static displayName = name;
-
-    render() {
-      const { children, ...props } = this.props;
-      delete props.active;
-      return (
-        <PageItem {...props}>
-          <span aria-hidden="true">{children || defaultValue}</span>
-          <span className="sr-only">{label}</span>
-        </PageItem>
-      );
-    }
+  return ({ children, ...props }) => {
+    return (
+      <PageItem {...props}>
+        <span aria-hidden="true">{children || defaultValue}</span>
+        <span className="sr-only">{label}</span>
+      </PageItem>
+    );
   };
```

Comment:
**issue:** for nice-ness in dev tools, we need to assign `displayName`. something like the below (maybe with a better interim function name than `PageItemButton`)

```suggestion
  function PageItemButton ({ children, ...props }) {
    return (
      <PageItem {...props}>
        <span aria-hidden="true">{children || defaultValue}</span>
        <span className="sr-only">{label}</span>
      </PageItem>
    );
  }

  PageItemButton.displayName = name;

  return PageItemButton;
```

Reply:
Yes, changes done ! thanks !